wls-gui-wahllokalsystem/src/views/auszaehlung/ErfassungStimmzettelView.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <the-ergebnisermittlung-stimmzettelumschlaege-card
+    :title="title"
+    :wahl-id="wahlID"
+    :use-time="!isUWB"
+  />
+</template>
+
+<script setup lang="ts">
+import { storeToRefs } from "pinia";
+import { computed, watch } from "vue";
+import { useRoute, useRouter } from "vue-router";
+
+import TheErgebnisermittlungStimmzettelumschlaegeCard from "@/components/ergebnisermittlung/TheErgebnisermittlungStimmzettelumschlaegeCard.vue";
+import { ROUTE_AUSZAEHLUNG_STIMMZETTEL } from "@/constants.ts";
+import { useUserStore } from "@/stores/userStore.ts";
+import { useWahlenStore } from "@/stores/wahlenStore.ts";
+
+const route = useRoute();
+const router = useRouter();
+const { isUWB } = storeToRefs(useUserStore());
+const { getWahlOrUndefinedById } = useWahlenStore();
+
+const wahlID = computed(() => route.params.wahlId as string);
+
+const title = computed(() => {
+  return isUWB.value
+    ? "Wahlurne öffnen und Stimmzettel zählen"
+    : "Wahlurne öffnen und Stimmzettelumschläge zählen";
+});
+
+watch(
+  () => route.params.wahlId,
+  (newId) => {
+    const wahl = getWahlOrUndefinedById(newId as string);
+
+    if (wahl) {
+      router.push({
+        name: ROUTE_AUSZAEHLUNG_STIMMZETTEL,
+        params: { wahlId: String(wahl.wahlID) },
+      });
+    }
+  }
```

Comment:
Wenn es zu der ID keine Wahl gibt würde ich auf die Error-Seite gehen

```suggestion
  () => wahlID.value,
  (wahlID) => {
    const wahl = getWahlOrUndefinedById(wahlID);

    if (!wahl) {
      router.push({
        name: EXAMPLE_ROUTES_NOTFOUND,
      });
    }
  }
```

Reply:
[06d3f50](https://github.com/it-at-m/Wahllokalsystem/pull/1802/commits/06d3f50aa18e6140a07efc27ec425c98b9f503b8)